mscore/data/shortcuts.xml

```diff
@@ -527,7 +527,7 @@
     <seq>F11</seq>
     </SC>
   <SC>
-    <key>toggle-navigator</key>
+    <key>toggle-timeline</key>
```

Comment:
For now, the timeline shortcut will be the navigator shortcut. Meaning navigator remains (for now).